Dockerfile.monitor

```diff
@@ -5,7 +5,7 @@ COPY ./VERSION /workdir/VERSION
 COPY ./.git /workdir/.git
 COPY ./src/runtime /workdir/src/runtime
 WORKDIR /workdir/src/runtime
-RUN CGO_ENABLED=1 GOFLAGS=-tags=strictfipsruntime make monitor
+RUN SKIP_GO_VERSION_CHECK=true CGO_ENABLED=1 GOFLAGS=-tags=strictfipsruntime make monitor
```

Comment:
Can you add a comment explaining the `SKIP_GO_VERSION_CHECK`? I know this is in the commit message, but having the same info here may be useful too.

Reply:
Done